lightning/src/util/events.rs

```diff
@@ -122,6 +122,13 @@ pub enum Event {
 		/// Note that this serves as a payment receipt, if you wish to have such a thing, you must
 		/// store it somehow!
 		payment_preimage: PaymentPreimage,
+		/// The hash which was given to [`ChannelManager::send_payment`].
+		/// When the `PaymentSent` message comes from an older library version,
+		/// the payment hash will be missing,
+		/// in which case the value of this field will be `None`.
```

Comment:
nit: 100char lines

Reply:
Hmmmm? What line are you exactly complaining about?

The only one that exceeds 100 chars is the last of these:

```
		/// The hash which was given to [`ChannelManager::send_payment`].
		/// When the `PaymentSent` message comes from an older library version,
		/// the payment hash will be missing,
		/// in which case the value of this field will be `None`.
		///
		/// [`ChannelManager::send_payment`]: crate::ln::channelmanager::ChannelManager::send_payment
```

But, there are plenty of those earlier in the source file

Reply:
This is the diff I'm looking for:
```
diff --git a/lightning/src/util/events.rs b/lightning/src/util/events.rs
index 03553651..cc54f9b1 100644
--- a/lightning/src/util/events.rs
+++ b/lightning/src/util/events.rs
@@ -122,10 +122,9 @@ pub enum Event {
                /// Note that this serves as a payment receipt, if you wish to have such a thing, you must
                /// store it somehow!
                payment_preimage: PaymentPreimage,
-               /// The hash which was given to [`ChannelManager::send_payment`].
-               /// When the `PaymentSent` message comes from an older library version,
-               /// the payment hash will be missing,
-               /// in which case the value of this field will be `None`.
+               /// The hash which was given to [`ChannelManager::send_payment`]. When the `PaymentSent` message
+               /// comes from an older library version, the payment hash will be missing, in which case the
+               /// value of this field will be `None`.
                ///
                /// [`ChannelManager::send_payment`]: crate::ln::channelmanager::ChannelManager::send_payment
                payment_hash: Option<PaymentHash>,
```

So we want lines to not be too short *or* too long.

Reply:
I'm sorry but that's not really very helpful feedback. You might as well the exact format that you want :\

Reply:
Well, the idea is that we want to fit as much text as is reasonable onto the page (hence avoiding too-short lines) in while also allowing multiple terminals side-by-side (hence the line length limit). Vim makes this super easy. Not sure how to be more helpful.

That said, it was a nit so you can ignore it if you really want.